deploy/README.md

```diff
@@ -0,0 +1,29 @@
+# Hydrant - deploy steps
+
+This file, and other contents of this folder, should be mirrored in `/afs/sipb/project/hydrant`. This folder is called our *locker*.
+
+If you're in the `sipb-hydrant` list, you can login to Athena and do `ssh hydrant@scripts`. You'll sign in to the Scripts account, whose home folder `~` is `/afs/sipb/project/hydrant`. In particular:
+
+- `~/README.md` links to `deploy/README.md`.
+- `~/cron_scripts` links to `deploy/cron_scripts`.
+- `~/web_scripts/notify_build.py` links to `deploy/web_scripts/notify_build.py`.
+  - Nothing else in `~/web_scripts` should be in the repo.
+- Nothing in `~/ci_secrets` should be in the repo.
+  - It has a `README` file, though. Read it in the locker!
```

Comment:
TODO: is this a good idea?

an argument against might be that maybe it's not prudent to expose all of our configuration to the world. then again, there aren't any secrets here.

if we think it's good to version control these publicly, we should make the symlinks then.

Reply:
The one big thing I can think of is if someone finds some kind of exploit for overloading `notify_build.py` or similar despite webhook authentication (e.g. by DOSing the endpoint); I do agree that they need some form of version control. (Maybe symlink notify_build.py to a secret hash and feed that to the hooks API?)

Reply:
after thinking about this i think this isn't a big deal, we can change it if necessary